config/environment.js

```diff
@@ -4,6 +4,8 @@
 module.exports = function (environment) {
   const pkg = require('../package.json');
   let deployTarget = process.env.DEPLOY_TARGET;
+  // Bring in the environment variable - test/stage/development only.
+  let minPrefixesAvailable = ((typeof process.env.MIN_PREFIXES_AVAILABLE === 'undefined') || (process.env.MIN_PREFIXES_AVAILABLE == "")) ? 50 : process.env.MIN_PREFIXES_AVAILABLE;
```

Comment:
I think this default to 50 is a great idea. 
just a clarifying question. why is it declared twice ? ( it's also in app/services/prefixes.js)

Reply:
Will fix this.